src/Sly/NotificationPusher/Adapter/Gcm.php

```diff
@@ -138,11 +138,11 @@ public function getServiceMessageFromOrigin(array $tokens, BaseOptionedModel $me
     public function getDefinedParameters()
     {
         return [
-            'collapse_key',
```

Comment:
@theluk, are you sure here? Please check https://github.com/Ph3nol/NotificationPusher/issues/144
Also, we had another 2 issues where people proposed to do opposite renaming from camelCase to under_score.